recipes/cgal/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "4.8.1" %}
+
+package:
+  name: cgal
+  version: {{ version }}
+
+source:
+  fn: CGAL-{{ version }}.zip
+  url: https://github.com/CGAL/cgal/releases/download/releases%2FCGAL-{{ version }}/CGAL-{{ version }}.zip
+  md5: 2780a9c046cbbf26838b8275d2719d5c
+
+build:
+  number: 0
+# fails with mpir as replacement of gmp on windows
+  skip: true  # [win]
```

Comment:
So Windows is not working? Can we remove any windows logic (`bld.bat` etc) and then add them as a PR on the feedstock to keep the recipe clean?